csrc/scheduler/matmul.cpp

```diff
@@ -57,6 +57,18 @@ void moveInnerBroadcastLeft(TensorView* tv, int number_of_inner_pos = 3) {
   tv->reorder(order_map);
 }
 
+// Utility to check concrete static size:
+auto check_concrete_static_dim = [](IterDomain* id) {
```

Comment:
A side question, is there some advantage of having alpha object in the anonymous namespace over function with inline keyword?

Reply:
actually I don't know how the decision is made but we are using a lot of  anonymous namespaces in this project. Maybe becase these are long internal functions.